streams/src/test/java/org/apache/kafka/streams/processor/internals/StoreChangelogReaderTest.java

```diff
@@ -51,7 +52,7 @@
     private final CompositeRestoreListener restoreListener = new CompositeRestoreListener(callback);
     private final MockConsumer<byte[], byte[]> consumer = new MockConsumer<>(OffsetResetStrategy.EARLIEST);
     private final StateRestoreListener stateRestoreListener = new MockStateRestoreListener();
-    private final StoreChangelogReader changelogReader = new StoreChangelogReader(consumer, stateRestoreListener);
+    private final StoreChangelogReader changelogReader = new StoreChangelogReader(consumer, stateRestoreListener, new LogContext("test-reader "));
```

Comment:
can we just extract the `LogContext` to a variable or field and re-use it below?